src/Internal/Contract/Monad.purs

```diff
@@ -197,7 +187,7 @@ mkContractEnv params = do
     b1 <- parallel do
       backend <- buildBackend logger params.backendParams
       -- Use the default backend to fetch ledger constants
```

Comment:
Comment seems redundant

Reply:
But we still use the default backend, not the optional one